src/db/migrations/20201213023900-update-product.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const vectorName = '_search';
+
+const searchObjects = {
+  Products: ['name', 'description'],
+  Stalls: ['name', 'description'],
+  HawkerCentres: ['name'],
+  Regions: ['name'],
+};
+
+module.exports = {
+  up: queryInterface =>
+    queryInterface.sequelize.transaction(t =>
+      Promise.all(
+        Object.keys(searchObjects).map(table =>
```

Comment:
nitpick: Could use `Object.entries` since need to access the values.
`Object.entries(searchObjects).map((table, fields) => ... )`

Reply:
Not tested yet but this should work

Reply:
I'll leave it as it affects the formatting of the SQL query.